x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -27,8 +27,12 @@ import { findingsNavigation } from '../../../common/navigation/constants';
 import { useUrlQuery } from '../../../common/hooks/use_url_query';
 import { ErrorCallout } from '../layout/error_callout';
 
-export const getDefaultQuery = (): FindingsBaseURLQuery & FindingsGroupByNoneQuery => ({
-  ...getDefaultUrlQuery(),
+export const getDefaultQuery = ({
+  query,
+  filters,
+}: FindingsBaseURLQuery): FindingsBaseURLQuery & FindingsGroupByNoneQuery => ({
+  query,
+  filters,
   sort: { field: '@timestamp', direction: 'desc' },
   pageIndex: 0,
   pageSize: 10,
```

Comment:
this is where we plug our injected app filters/query to the current page's default query getter. 

that getter is already what's used to set up the current page's URL and state, now it includes the current filters/query to persist between navigations